primers/spatial-computing/README.md

```diff
@@ -1,80 +1,36 @@
-# Spatial Computing: An Introduction to the Subject
-
-# Contents
-
-[Introduction to Spatial Computing](#_Toc149764170)
-- [Definition & Overview](#_Toc149764171)
-- [Merging Digital & Physical Worlds](#_Toc149764172)
-- [Hardware](#_Toc149764173)
-- [Emerging Current Trends](#_Toc149764174)
-- [What comes next?](#_Toc149764175)
-
-[A brief history of Spatial Computing](#_Toc149764176)
-- [Early Developments](#_Toc149764177)
-- [Key Milestones](#_Toc149764178)
-- [Current State & Future Directions](#_Toc149764179)
-
-[Types of Reality & their Use Cases](#_Toc149764180)
-- [Augmented Reality](#_Toc149764181)
-- [Virtual Reality](#_Toc149764182)
-- [Mixed Reality](#_Toc149764183)
-
-[Applicability in Financial Services](#_Toc149764184)
-- [Banking](#_Toc149764185)
-- [Investment](#_Toc149764186)
-- [Customer Service](#_Toc149764187)
-- [Operations](#_Toc149764188)
-
-[Tools & Development Platforms](#_Toc149764189)
-- [Software Development Kits (SDKs)](#_Toc149764190)
-- [Hardware & Devices](#_Toc149764191)
-- [Development Environments](#_Toc149764192)
-- [Content Creation Tools](#_Toc149764193)
-- [Cross-Platform Development](#_Toc149764194)
-
-[Challenges & Considerations](#_Toc149764195)
-- [Privacy & Security Concerns](#_Toc149764196)
-- [User Adoption Challenges](#_Toc149764197)
-- [Regulatory Compliance](#_Toc149764198)
-- [Integration with Legacy Systems](#_Toc149764199)
-- [Data Accuracy and Reliability](#_Toc149764200)
-- [Cost and Resource Allocation](#_Toc149764201)
-- [Ethical Considerations](#_Toc149764202)
-- [Scalability and Performance](#_Toc149764203)
-- [Human-Centric Design](#_Toc149764204)
-- [Training and Skill Development](#_Toc149764205)
-
-***
-
-# [Introduction to Spatial Computing](#_Toc149764170)
-
-## [Definition & Overview](#_Toc149764171)
+---
+title: "Spatial Computing"
+---
+
+## Introduction to Spatial Computing
+
+### Definition & Overview
```

Comment:
### for Introduction  and #### for Definition & Overview in other two primers

Reply:
Good spot!